MapboxNavigation/NetworkConfiguartion.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+class NetworkConfiguration : NSObject{
```

Comment:
Per mapbox/mapbox-gl-native#12417, it seems like the `hasChinaAPIBaseURL` property should go on an extension of MGLAccountManager instead of this singleton.